src/lib/network/clients/save_network.rb

```diff
@@ -77,12 +79,12 @@ def CopyConfiguredNetworkFiles
         file = receipt[:dir] + receipt[:file]
         adjust_for_network_disks(file) if file.include?("ifcfg-")
 
-        copy_from = String.Quote(file)
-        copy_to = String.Quote(inst_dir + receipt[:dir])
+        copy_from = file
+        copy_to = inst_dir + receipt[:dir]
 
         log.info("Copying #{copy_from} to #{copy_to}")
 
-        cmd = "cp " << copy_from << " " << copy_to
+        cmd = "cp " << copy_from.shellescape << " " << copy_to.shellescape
```

Comment:
Please use `+` or `#{}` to join strings, mutating with `<<` will break once Ruby enables frozen literals.